### DISCUSSION TITLE
StreamAPI

### DISCUSSION THREAD

**Discussion by jct94** [fastapi/fastapi#7047] [Questions] [answered]
Hello Guys, I have a question regarding FastAPI,is that possible to render a stream of results from a ML model ? My question is, I have built a stream listener on twitter, and would like to infer sentiment of each tweets in real time, then expose result to the API so my front end folks can fetch it : Is that feasible ? relevant ? idiot ? should I stop programming for ever ?

**Comment by includeamin** [comment, marked as answer]
` Should I stop programming for ever` nooo :)) keep going.
 it can be done with [FastApi's Websocket](https://fastapi.tiangolo.com/advanced/websockets/) or [starlette's channel](https://github.com/encode/broadcaster) or 
[python socketio](https://python-socketio.readthedocs.io/en/latest/).

**Comment by jct94** [comment]
Thanks ! Will double check that

**Comment by tiangolo** [comment]
Thanks for the help here @includeamin ! :clap: :bow: 

Thanks for reporting back and closing the issue @jct94  :+1:
